language-server/src/dotty/tools/languageserver/worksheet/Worksheet.scala

```diff
@@ -22,15 +22,15 @@ object Worksheet {
    */
   def run(tree: SourceTree,
           treeLock: Object,
-          sendMessage: (Int, String) => Unit,
+          sendMessage: (SourcePosition, String) => Unit,
```

Comment:
It is enough to have a `Span` here.

Reply:
Actually we need a SourcePosition to call `DottyLanguageServer.range`